tests/test262-esnext-excludelist.xml

```diff
@@ -9668,7 +9666,6 @@
   <test id="built-ins/RegExp/prototype/sticky/cross-realm.js"><reason></reason></test>
   <test id="built-ins/RegExp/prototype/unicode/cross-realm.js"><reason></reason></test>
   <test id="built-ins/SharedArrayBuffer/proto-from-ctor-realm.js"><reason></reason></test>
-  <test id="built-ins/String/proto-from-ctor-realm.js"><reason></reason></test>
```

Comment:
Only realm tests are fixed? No normal tests for these?

Reply:
Sadly, there is no new target test for boolean, string and number object.